Automattic/wp-calypso
# Plugins: Replace the hardcoded featured plugin list with results from the wpcom `/plugins/featured` endpoint.

Replace the hardcoded featured plugin list with results from the wpcom `/plugins/featured` endpoint. The endpoint simply serves the same curated list that's being removed, but by moving it to an endpoint, the data can be used by the mobile apps.

![screen-shot-2018-01-15-at-3-22-19-pm](https://user-images.githubusercontent.com/349751/35072515-bf822394-fb99-11e7-83d5-d1493fb52969.png)

This section still uses Flux stores, so knowing a refactor to using the Redux store should happen in the future, I've gone with a simple solution. Note also that I've left tests as they are.

**Testing**
* Switch to this branch, and load `/plugins`.
* Verify that the top Featured section displays plugins, without any errors.
* The Network panel should show a request to `https://public-api.wordpress.com/wpcom/v2/plugins/featured?_envelope=1` with the plugin data received.

See: p9c7Jx-mx-p2
